popup/date-since.md

```diff
@@ -0,0 +1,51 @@
+
+# Add a date since last update field to a pop-up
+
+[![time_popup](./images/timing.PNG)]
+
+This example shows how to add days since the last update to a pop-up
+
+## Use cases
+
+This is designed specifically for the popup profile. These set of expressions allow you to represent the number of days since a point or polygon has been udpated.
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+## Expression Template
+
+Create an expression to identify the date field and find the 
+
+```js
+//get the time currently
+var timeNow = Date()
+//get the time field form your feature layer
+var survey = $feature["date_survey1"]
+//get time in hrs, days and use datediff to get the differnce betwen the time now and the time when the layer was updated
+var datehrs = Round(DateDiff(timeNow, survey, 'hours'),1)
+var datedays = Round(DateDiff(timeNow, survey, 'days'),0)
```

Comment:
I think the params in the DateDiff function should be flipped. The `survey` date is usually the older date, correct? Otherwise, you'll always get negative values here. I ran a quick test in the playground and that seemed to confirm this behavior. I would modify to: 

```suggestion
var datedays = Round(DateDiff(survey, timeNow, 'days'),0)
```

Documentation here - https://developers.arcgis.com/arcade/function-reference/date_functions/#datediff